src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -94,6 +95,8 @@ class RtpReceiverImpl @JvmOverloads constructor(
     private val inputTreeRoot: Node
     private val incomingPacketQueue =
             PacketInfoQueue("rtp-receiver-incoming-packet-queue", executor, this::handleIncomingPacket, Config.queueSize())
+    private val incomingPacketQueueStats = QueueStatistics(incomingPacketQueue)
```

Comment:
I've usually done this via:
```
private val incomingPacketQueueStats = QueueStatistics(incomingPacketQueue).also {
    incomingPacketQueue.setObserver(it)
}
```

To avoid multiple `init`s nested within member definitions, but I don't feel strongly.